jsonapi.gemspec

```diff
@@ -0,0 +1,16 @@
+version = File.expand_path('../JSONAPI_VERSION', __FILE__).strip
```

Comment:
Well here, the base `jsonapi` gem would have to be re-released each time either of the subgems gets a new release, so as a quick hack, I thought I'd keep the 3 in sync at the beginning. Think of it like this: there is fundamentally one gem, but as a convenience you can depend on only a part of it. That's the versioning scheme Rails uses for its various components (activesupport, actionpack, etc.).